docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
-. [[ref:10]] https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/solution/dcat-application-profile-data-portals-europe/distribution/dcat-ap-v11-pdf
-. [[ref:11]] https://www.dublincore.org/specifications/dublin-core/profile-guidelines/
-. [[ref:12]] ePO53
-. [[ref:13]] ePO63
-. [[ref:14]] ePO61
-. [[ref:15]] SEMIC toolchain  - https://github.com/SEMICeu [???]
-. [[ref:16]] model2owl - https://github.com/OP-TED/model2owl [???]
-. [[ref:17]] OSLO toolchain - https://github.com/Informatievlaanderen/OSLO-publicationenvironment-template [???]
-. [[ref:18]] ePO UML architecture, sec 4.1
-. [[ref:19]] OSLO spec, sec 3.1.2
-. [[ref:20]] Semic Draft style guide 3.2
-. [[ref:21]] skos [???]
-. [[ref:22]] ePO UML conventions, sec 3.1
-. [[ref:23]] ePO UML conventions, sec 3.6
-. [[ref:24]] ePO UML conventions, sec 3.7
-. [[ref:25]] https://github.com/SEMICeu/OpenGuidelines/blob/main/Principles%20for%20creating%20good%20definitions_v1.00.pdf
-. [[ref:26]] ePO UML conventions, sec 4.2
-. [[ref:27]] ePO UML conventions, sec 4.1
-. [[ref:28]] https://www.omg.org/spec/ODM/1.1/PDF
-. [[ref:29]] UML interpretation for SEMIC toolchain
-. [[ref:30]] EPO conventions
-. [[ref:31]] EPO transformation rules
-. [[ref:32]] ePO2 [???]
-. [[ref:33]] ePO30 [???]
-. [[ref:34]] ePO5 [???]
-. [[ref:35]] ePO24 [???]
-. [[ref:36]] ePO UML conventions, sec 3.2
-. [[ref:37]] OSLO spec, sec 3.2.20
-. [[ref:38]] https://www.w3.org/TR/REC-xml-names/
-. [[ref:39]] ePO28 [???]
-. [[ref:40]] https://www.w3.org/TR/turtle/#sec-iri
-. [[ref:41]] OSLO spec, sec 3.1.7
-. [[ref:42]] OSLO spec, sec 3.2.3
-. [[ref:43]] ePO21 [???]
-. [[ref:44]] ePO14 [???]
-. [[ref:45]] OSLO spec, sec 3.2.6
-. [[ref:46]] ePO UML conventions, sec 4.4
-. [[ref:47]] ePO - https://github.com/OP-TED/ePO
-. [[ref:48]] OSLO spec, sec 3.2.12
-. [[ref:49]] ePO UML conventions, sec 4.5
-. [[ref:50]] ePO UML conventions, sec 4.7
-. [[ref:51]] OSLO spec, sec 3.2.2
-. [[ref:52]] OSLO spec, sec 3.2.15
-. [[ref:53]] OSLO spec, sec 3.2.16
-. [[ref:54]] OSLO spec, sec 3.2.22
-. [[ref:55]] OSLO spec, sec 3.2.17
-. [[ref:56]] ePO UML conventions, sec 4.3
-. [[ref:57]] OSLO spec, sec 3.3.4
-. [[ref:58]] OSLO spec, sec 3.3.1
-. [[ref:59]] OSLO spec, sec 3.3.5
-. [[ref:60]] OSLO spec, sec 3.3.3
-. [[ref:61]] OSLO spec, sec 3.3.2
-. [[ref:62]] OSLO spec, sec 3.2.5
-. [[ref:63]] OSLO spec, sec 3.1.11
-. [[ref:64]] Lemon
+. [[ref:1]] Codreanu, D. (2012) 10 rules for persistent uris, Joinup. Available at: https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/document/10-rules-persistent-uris.
+. [[ref:2]] A. Miles and S. Bechhofer. SKOS simple knowledge organization system reference. W3C recommendation, W3C, Aug. 2009.
+http://www.w3.org/TR/2009/REC-skos-reference-20090818/.
+. [[ref:3]] E. Costetchi. eProcurement uml conceptual model to owl ontology transformation. Recommendation, Publications Office of the European Union, April
```

Comment:
Publication *Year* is very important. We need to provide it each reference. (if no publication is available, then access date, in the case of websites)

Reply:
hopefully, I did not forget anything (done)